src/test/java/org/broadinstitute/hellbender/tools/walkers/ValidateVariantsIntegrationTest.java

```diff
@@ -127,7 +127,8 @@ public void testBadID2_OKif_noDBSNPArgument() throws IOException {
     public void testBadID2_OKif_notInDBSNP() throws IOException {
         final IntegrationTestSpec spec = new IntegrationTestSpec(
                 baseTestString(false, "validationExampleRSIDonPositionNotInDBSNP.vcf", false, IDS) + " --dbsnp " + hg19_chr1_1M_dbSNP,
-                Collections.emptyList()
+                0,
+                UserException.LexicographicallySortedSequenceDictionary.class
```

Comment:
These tests in `ValidateVariantsIntegrationTest` need to be refactored -- updating the expected exception like this invalidates what they are trying to test. Eg., this test is checking whether a "bad id is ok if not in DBSNP", and so should not be expecting an exception, but now it does.